libraries/androidutils/src/main/kotlin/io/element/android/libraries/androidutils/filesize/AndroidFileSizeFormatter.kt

```diff
@@ -40,6 +41,6 @@ class AndroidFileSizeFormatter @Inject constructor(
             Formatter.formatShortFileSize(context, normalizedSize)
         } else {
             Formatter.formatFileSize(context, normalizedSize)
-        }
+        }.replace("kB", "KB")
```

Comment:
Why this replacement? It seems that `kB` is more correct than `KB`, since `k` is the official shortcut for `kilo` in the international system of units.

Reply:
In the end did we decide on whether to use 1024 or 1000? I think it was 1000?